src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -137,6 +137,9 @@ internal sealed class SolutionRestoreJob : ISolutionRestoreJob
                 {
                     await RestoreAsync(request.ForceRestore, request.RestoreSource, token);
                 }
+                catch (OperationCanceledException) when (token.IsCancellationRequested)
```

Comment:
This change to swallow `OperationCancelledException` is simply to avoid polluting activity log with exception stacks when cancellation is requested. There's no other change in behavior rather than that nuance.